src/main/java/vazkii/botania/common/block/mana/BlockForestDrum.java

```diff
@@ -149,9 +149,11 @@ else if(variant == DrumVariant.CANOPY)
 						if(itemstack != null && itemstack.getItem() == Items.BUCKET && !world.isRemote) {
 							while(itemstack.stackSize > 0) {
 								EntityItem ent = entity.entityDropItem(new ItemStack(Items.MILK_BUCKET), 1.0F);
-								ent.motionY += world.rand.nextFloat() * 0.05F;
-								ent.motionX += (world.rand.nextFloat() - world.rand.nextFloat()) * 0.1F;
-								ent.motionZ += (world.rand.nextFloat() - world.rand.nextFloat()) * 0.1F;
+								if(ent != null) { // Shouldn't be given that we specified the item to drop, but let's be safe anyway
```

Comment:
remove the comment since it's clear we check bc entityDropItem is nullable

looks good besides this + extra commits